server/setup.py

```diff
@@ -46,7 +46,7 @@
         'torch',
         'regex',
         'torchvision',
-        'jina>=3.8.0',
+        'jina>=3.11.0',
         'prometheus-client',
```

Comment:
can we remove `prometheus-client` now?

Reply:
Not yet because Jina Cloud doesn't support the new backends for tracing and metrics.l